### PR TITLE
Switch to latest v5 release of release-drafter

### PR DESCRIPTION
**Pull request by jetersen**
👋 Hi we recently fixed some of the bugs you reported over at release-drafter. So here I am updating your action to get those patches.

https://github.com/release-drafter/release-drafter/pull/412
https://github.com/release-drafter/release-drafter/pull/424

### CODE REVIEW

**Comment by rnorth** [comment]
Awesome, thank you @jetersen!

Just wondering - is there any specific need to specify the version as `v5` rather than `v5.9.0` (which seems the be the latest)?

I think I'd prefer to do this if we can so that we don't have a floating dependency version.

**Comment by jetersen** [comment]
v5 points to the latest version of major version 5.
This is actually based on the recommended versioning for GitHub actions.

https://help.github.com/en/actions/creating-actions/about-actions#using-release-management-for-actions

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -9,6 +9,6 @@ jobs:
   update_release_draft:
     runs-on: ubuntu-latest
     steps:
-      - uses: release-drafter/release-drafter@v5.2.0
+      - uses: release-drafter/release-drafter@v5
```
Comment (kiview) [inline .github/workflows/release-drafter.yml:12]:
```suggestion
      - uses: release-drafter/release-drafter@v5.9.0
```

I think we would prefer the fixed version here 🙂

**Comment by jetersen** [comment]
Might be worthwhile waiting for the next release with https://github.com/release-drafter/release-drafter/pull/546 incoming 😆

**Comment by kiview** [comment]
I think @rnorth mentioned a Dependabot feature for updating actions 🙂

**Pull request merged**